pkg/sloop/webserver/webfiles/filter.js

```diff
@@ -16,7 +16,7 @@ function getUrlVars() {
 function getUrlParam(parameter, defaultvalue){
     var urlparameter = defaultvalue;
     if(window.location.href.indexOf(parameter) > -1){
-        urlparameter = getUrlVars()[parameter];
+        urlparameter = decodeURIComponent(getUrlVars()[parameter]);
```

Comment:
Is this related to table change?

Reply:
Yes its for the same page. The text box in the page gets encoded every time we submit the page